handbook/communication/decisions.md

```diff
@@ -1,23 +1,82 @@
+
+<a name="approvers"></a>**Approver(s)**: As short a list as possible of individuals who must approve a recommendation before it can be adopted. This role is slightly different from the [decider](#decider) role—they hold the right to veto, not to choose. Having more than 1 or 2 approvers is a sign of a poorly scoped decision, or a symptom of unclear organizational ownership/responsibilities.
+
+<a name="inputs"></a>**Input provider(s)**: Anyone who can provide input (e.g. historical context, personal experience, cross-functional points of view, etc.) into a decision making process. While this list should be kept short by default to expedite decisions, at Sourcegraph everyone is welcome to provide input.
+
+<a name="decider"></a>**Decider**: the one individual who ultimately makes the decision. This person must be aware of and bought into their role.
+
+### How to use these roles when making a decision
+
+For large, complex decisions, it can be helpful to build a “RAID matrix”: a table with decision-making steps in the rows and names of the teammates that are involved in the columns. In each cell a letter or multiple letters (“R”, “A”, “I”, and “D”) can be written to indicate roles.
+
+However, in most cases this exercise will be overkill. Simply making the [approver](#approvers) and [decider](#decider) roles clear for big decisions up-front is usually sufficient.
+
+Our [RFCs](rfcs/index.md) encode several of these roles already, with the “authors” serving as [recommenders](#recommenders) (and sometimes [deciders](#decider)) and the “approvers” containing a mix of [approvers](#approvers) and [input providers](#inputs) (and sometimes [deciders](#decider)).
+
+## So who decides?
+
+One of the hardest parts of making a decision (that is rarely addressed by a framework) is how to decide who decides!
+
+The key principle is that every decision should be made by the individual who is most directly responsible for the **execution** and **results** of the decision.
+
+That individual’s manager may feel responsible for the results, but if they’re not owning the execution, they are likely not the right [decider](#decider).
+
+Our high-agency culture is only successful if individuals across the organization are empowered to make decisions. If you or someone on your team is rarely making decisions, that’s a sign of organizational failure. Managers at Sourcegraph are celebrated for delegating decision-making by empowering their teams.
+
+### Examples
+
+While the principle above is clear, how to apply it can be murky. Examples below are helpful to illustrate it.
+
+#### Changing our team’s OKR for this quarter
+
+- Because the team itself is responsible for the execution and results of their own OKRs, the [decider](#decider) should be clear: the team manager must make the decision to change an OKR.
+
+#### Choosing software to use for expense reimbursements
```

Comment:
Clarify that this is reimbursement software like Expensify not about choosing a tool for which you will be reimbursed